.github/workflows/cli-release.yml

```diff
@@ -0,0 +1,83 @@
+name: CLI Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      draft-release:
+        description: 'Draft release?'
+        required: false
+        type: boolean
+        default: true
+      prerelease:
+        description: 'Prerelease?'
+        required: false
+        type: boolean
+        default: true
+      npm-version:
+        description: 'NPM version bump?'
+        required: false
+        type: choice
+        options: 
+          - major
+          - minor
+          - patch
+        default: 'patch'
+
+  # schedule:
+  #   - cron: '0 12 * * 2'  # Runs every Tuesday at 12:00 PM (noon) ET
```

Comment:
Is this schedule something we're planning on doing? Or just an example?

Reply:
yes, I think the team is planning to do the release on schedule, but don't have a decision yet. Putting a comment here to remind myself and the team to bring this up again after this script is merged.